Doc/whatsnew/3.12.rst

```diff
@@ -856,6 +856,19 @@ although there is currently no date scheduled for their removal.
   :keyword:`for`, :keyword:`if`, :keyword:`in`, :keyword:`is` and :keyword:`or`.
   In a future release it will be changed to a syntax error. (:gh:`87999`)
 
+* The following :mod:`ast` features have been deprecated in documentation since
+  Python 3.8, now cause a :exc:`DeprecationWarning` to be emitted at runtime
+  when they are accessed or used, and may be removed in a future version of Python:
```

Comment:
We could also just set 3.14 as the removal date. @isidentical do you have any thoughts on that?